packs/salt/actions/local.py

```diff
@@ -14,7 +14,10 @@ def run(self, **kwargs):
             st2 run salt.client module=pkg.install \
                     kwargs='{"pkgs":["git","httpd"]}'
         '''
-        self.generate_package(cmd=cmd)
+        # TODO: This is broken, fix it. I temporary disabled it to avoid pylint
```

Comment:
@jcockhren Can you please have a look?

The action appears broken, but I don't have enough context to fix it (what to do with args and kwargs parameters which are passed to the action) so I rather just commented it for now.

It would also be better if we get rid of "kwargs abuse" and explicitly declare arguments in the method signature.